hack/build.sh

```diff
@@ -63,6 +63,19 @@ run() {
     exit 0
   fi
 
+# platform mode: Only to build target platform for cross-compilation and maybe run test
+  if $(has_flag --platform -p); then
+  # Extract GOOS and GOARCH from command-line arguments
+    GOOS="${ARGS[1]}"
+    GOARCH="${ARGS[2]}"
+    go_build_with_goos_goarch "$GOOS" "$GOARCH"
+
+    if $(has_flag --test -t); then
+       go_test
+    fi
```

Comment:
What's the reason to keep this if statement inside nested inside if `has_flag`?

Reply:
I Keep it like other functions, just to be consistent with others.